scripts/generate-schema.js

```diff
@@ -2,7 +2,7 @@
 
 "use strict";
 
-if (process.env.NODE_ENV === "test") {
+if (process.env.NODE_ENV === "test" || process.env.CI) {
```

Comment:
What about using `if (requre.main === module)` as the condition ([more info on Stack Overflow](https://stackoverflow.com/a/6398335/5244995)), and inverting the two blocks?

This way, you can still `require()` the module outside of CI/tests.